### PR TITLE
Iter13

### PR DESCRIPTION
**Pull request by DimaKoz**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -41,4 +41,24 @@ test8:
...
+	rm -f ./tempfile*
+
+lnt:
+	# excluded 'paralleltest' by the reason - not now
+	# excluded 'wsl' by the reason - 'wsl' and 'gofumpt' fights between each other
+	# excluded 'gochecknoglobals' by reason - I need global variables sometimes
+	# excluded 'exhaustivestruct' - deprecated
+	# excluded 'depguard' - no need in it
+	# excluded 'ifshort' - deprecated
+	golangci-lint run -v --enable-all --disable gochecknoglobals --disable paralleltest --disable exhaustivestruct --disable depguard --disable wsl
```
Comment (arttet) [inline Makefile]: You need to use config for linters.

File: Makefile [hunk 2]
```diff
@@ -41,4 +41,24 @@ test8:
...
+
+
+fmt:
+	# to install it:
+	# go install mvdan.cc/gofumpt@latest
+	gofumpt -l -w .
```
Comment (arttet) [inline Makefile:58]: You can add `gofmt` either

**Review by arttet — CHANGES_REQUESTED** [review]
Please add indexes and migrations

File: Makefile (hunk 2 above)
Comment (DimaKoz) [reply Makefile:58]: done

File: Makefile (hunk 1 above)
Comment (DimaKoz) [reply Makefile]: done

**Pull request merged**